suite-native/transactions/src/components/TransactionDetail/TransactionDetailParametersSheet.tsx

```diff
@@ -23,6 +26,25 @@ const transactionIdStyle = prepareNativeStyle(_ => ({
     maxWidth: '72%',
 }));
 
+type TransactionParameter =
```

Comment:
I don't like idea of redefining all these types manually. All of these types should be already defined somewhere. For example for ETH you could do something like:

```ts
{
   eth: Array<keyof BlockbookTransaction['ethereumSpecific']>
}
```
And you should be able to find probably rest of other types somewhere.

Reply:
`TransactionParameter` type reworked in [717bf00](https://github.com/trezor/trezor-suite/pull/7933/commits/717bf00f4d717e813a046e6e4a18555222229742).